net.rpcs3.RPCS3.yaml

```diff
@@ -0,0 +1,159 @@
+id: "net.rpcs3.RPCS3"
```

Comment:
Do you really want to keep all that quoting? Isn't YAML all about being less noisy?

Reply:
I prefer to be explicit on the types of values. I've had enough bad experiences with this with YAML.